Create tests to evaluate the number of false positives and false negatives (#222)
DataDog/guarddog: README.md

# Scan a local package
guarddog pypi scan /tmp/triage.tar.gz

> # Scan a local directory

Reviewer: Can we be more specific about what we expect the directory to contain? (1 .tar.gz per package?)

Author: Done

Running unit tests against package metadata heuristics: `make test-metadata-rules` (tests are [here](https://github.com/DataDog/guarddog/tree/main/tests/analyzer/metadata)).

### Benchmarking

> Run guarddog on legit and malicious packages to determine the ratio of false positives / false negatives
> You can go [./tests/samples](./tests/samples)

Reviewer: ```suggestion
You can run GuardDog on legitimate and malicious packages to determine false positives and false negatives. See [./tests/samples](./tests/samples)
```
